### DISCUSSION TITLE
Anyone else not able to create a plot?

### DISCUSSION THREAD

**Discussion by misterskye** [earthlab-education/Earth-Analytics-2023-01-Intro#251] [Q&A02 Open Reproducible Science]
The code seems to be working, but I'm getting an error of some sort.   

/opt/conda/lib/python3.10/site-packages/holoviews/core/data/pandas.py:39: FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`
  return dataset.data.dtypes[idx].type

![image](https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/47870472/95ab6836-4883-45a7-b2c8-2ec847004e2d)

**Comment by edchanonline** [comment]
Hey @misterskye - 
That's just a warning that's not relevant. You need to assign your code with a variable. 
Then, in the following line, you have to type out the variable name explicitly in order for the Jupyter Notebook to display the content of the variable in the output cell. 

e.g.

```
plot = df.hvplot(...)
plot
```

**Reply by misterskye** [reply]
Thanks Ed!!!  Amazing how easy it can be :-)  
Why didn't Elsa need to assign it to a variable?

**Reply by eculler** [reply]
hm, not sure about that. But I have been getting the FutureWarning sometimes - I'll have to pay attention to whether this is what determines it.

A number of folks have also run into the problem where the plot just doesn't display. It does not seem to be dependent on saving/calling a variable. We also tried a couple of other things, like calling `display(my_plot)` at the end of the cell. The only thing so far that consistently fixes this problem is **committing and syncing all work** and then starting a new Codespace and deleting the old one.

A workaround is to use pandas plotting instead, which creates static images. It's very similar - you use `plot` instead of `hvplot` with the same dataframe and parameters:

```
df.plot(...)
```
